PrivateHeaders/XCTest/XCUIDevice.h

```diff
@@ -11,9 +11,16 @@
 // Since Xcode 10.2
 @property (readonly) id accessibilityInterface; // implements XCUIAccessibilityInterface
 @property (readonly) id eventSynthesizer; // implements XCUIEventSynthesizing
-@property(readonly) id screenDataSource; // @synthesize screenDataSource=_screenDataSource;
+@property (readonly) id screenDataSource; // @synthesize screenDataSource=_screenDataSource;
+
 - (_Bool)performDeviceEvent:(id)arg1 error:(id *)arg2;
 
+// Since Xcode 13
+// 1 - Light
+// 2 - Dark
+- (void)setAppearanceMode:(long long)arg1;
+- (long long)appearanceMode;
```

Comment:
It seems like `userInterfaceStyle` in device info should follow this value if available.
In a real device, this value is 0 -> automatic, 1 -> light, 2 -> dark.
(Not for this pr itself)